cypress-custom/integration/fee.test.ts

```diff
@@ -54,9 +54,53 @@ describe('Fetch and persist fee', () => {
 
   // TODO: not sure if it's easy to test this
   it('Re-fetched when it expires', () => {
+    const NETWORK = ChainId.RINKEBY.toString()
+    const TOKEN = 'ETH'
+    const KEY = 'redux_localstorage_simple_fee'
+    const VALUE = JSON.stringify({
+      [NETWORK]: {
+        [TOKEN]: {
+          fee: {
+            minimalFee: '777',
+            feeRatio: 777,
+            expirationDate: new Date(Date.now() - 200000).toISOString()
+          },
+          token: TOKEN
+        }
+      }
+    })
     // GIVEN: A fee is present in the local storage
-    // WHEN: When the fee quote expires, we refetch the fee
-    // THEN: We get another quote
+    // Set expiring fee in localStorage
+    cy.get<Storage>('@localStorage')
+      .then($storage => {
+        // set time in the past
+        $storage.setItem(KEY, VALUE)
```

Comment:
why separating the VALUE from the given? 

don't fix, I'll give a try on making this more slim in a bit